### PR TITLE
33Across Bid Adapter: Obtain display-related attributes 

### PR DESCRIPTION
**Pull request by carlosfelix**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Obtain display-related client-side attributes and clients hints from browsers that support it 

- contact email of the adapter’s maintainer: carlos.felix@33across.com
- [X] official adapter submission

### CODE REVIEW

**Comment by carlosfelix** [comment]
I've tried many different ways to reproduce the failing tests scenario that I am seeing in the CI, but no luck so far, all the specs are passing on my environment and other envs where I have tried.

**Comment by patmmccann** [comment]
We're not allowing adapters to access these api's, please modify core and access from core

https://github.com/prebid/Prebid.js/issues/8381

File: modules/33acrossBidAdapter.js [hunk 1]
```diff
@@ -61,6 +62,44 @@ function getTTXConfig() {
...
+function getStoredUaReducedData() {
+  try {
+    return JSON.parse(storage.getDataFromLocalStorage(UA_DATA_KEY));
+  } catch (err) {
+    return null;
+  }
+}
+
+function storeUaReducedData(uaData) {
+  storage.setDataInLocalStorage(UA_DATA_KEY, JSON.stringify(uaData));
```
Comment (patmmccann) [inline modules/33acrossBidAdapter.js:77]: why put this in storage?

**Review by patmmccann — CHANGES_REQUESTED** [review]
User agent client hints should be gathered by core

**Comment by patmmccann** [comment]
Update: discussed with 33across team on slack, updates pending

**Comment by macinjosh32** [comment]
Hi Pat,
We're only intending to collect client hints within our adapter on a temporary basis (through end of Q3) so we can quickly understand the state of various browsers' UA deprecation. We'll be happy to switch the access over to enrichmentFpdModule thereafter if we decide we need it past Q3. It looks like several other modules, such sspBC's adapter and Just ID's User ID sub-module (via an external script) are already collecting these on a more permanent basis, so we were not anticipating any concerns as long as standards are being fairly applied across modules. Placing the values in storage is to help facilitate unit tests and ensure these were available upon initial request since the Client Hints API is asynchronous. Thanks in advance for your understanding.

**Comment by patmmccann** [comment]
We noticed that as well, which is why we drew the line in the sand to make sure no more were added https://github.com/prebid/Prebid.js/issues/8381

**Comment by anthonyjl92** [comment]
Hi Pat, we removed the storage and client hints from this PR - are you able to review the remaining changes? Thanks!

**Review by patmmccann — APPROVED** [review]

**Pull request merged**